src/Digipolis.Delegation/Digipolis.Delegation.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>Toolbox for user delegation.</Description>
     <Authors>digipolis.be</Authors>
     <VersionPrefix>1.1.5</VersionPrefix>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
kon dit niet op 2.0 blijven, kwestie van backwards compatible te zijn met bestaande toolboxen?

Reply:
> @PeterBrion @erikseynaeve sample app bevat wel info over security. Zouden we dit dan niet beter private maken en bv. naar bitbucket pushen?

Als het niet mogelijk is om die sample app generieker op te zetten, lijkt me dit ook beter. Dan kan dit als een nieuwe repo onder hetzelfde project als auth-toolbox toegevoegd worden. Dit project kan dan als DEPRECATED gemarkeerd worden.